torchaudio/prototype/models/hdemucs.py

```diff
@@ -498,6 +493,16 @@ def _ispec(self, z, length=None):
         x = x[..., pad : pad + length]
         return x
 
+    def _pad1d(self, x: torch.Tensor, paddings: tp.Tuple[int, int], mode: str = "zero", value: float = 0.0):
+        """Wrapper around F.pad, in order for reflect padding to not break. Add extra zero padding around"""
```

Comment:
can you add a little more detail here? in which scenario does it break and how does this resolve it